enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -536,6 +540,10 @@ func (p *PebbleCache) deleteOrphanedFiles(quitChan chan struct{}) error {
 			default:
 			}
 
+			dupPartitionPattern := fmt.Sprintf("%s/%s", part.ID, part.ID)
+			pathHasDupPartitionID, err := regexp.MatchString(dupPartitionPattern, path)
```

Comment:
Do you need regexp here, would `strings.Contains` work just as well?

Reply:
Good call - don't know why I used regex

Reply:
AFAIK, nobody is using pebble on-prem yet.